maxun-core/src/interpret.ts

```
@@ -548,232 +548,277 @@ export default class Interpreter extends EventEmitter {
+
+    try {
+      while (true) {
+        // Reduced timeout for faster performance
+        await page.waitForLoadState('networkidle', { timeout: 10000 }).catch(() => {});
+        
+        switch (config.pagination.type) {
+          case 'scrollDown':
+            await page.evaluate(() => window.scrollTo(0, document.body.scrollHeight));
+            await page.waitForTimeout(2000);
+
+            const currentHeight = await page.evaluate(() => document.body.scrollHeight);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)